tests/subsys/dfu/dfu_target/suit/regenerate_manifests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Scrtipt to regenerate SUIT manifests for DFU target tests.
+# To use this script you must have the following tools installed:
+# - suit-generator
+# - zcbor
```

Comment:
we should not be requiring additional tools outside what we provide with the NCS Toolchain.

Reply:
Most of the SUIT tests require these both tools.

Reply:
AFAIK both of those tools are already a part of NCS Toolchain

Reply:
accepted. the wording just indicated it was not :wink: